### PR TITLE
Use GroundPrimitive for line

### PR DESCRIPTION
**Pull request by oterral**
It corresponds to the hack in Cesium.KmlDataSource.

### CODE REVIEW

File: src/featureconverter.js [hunk 1]
```diff
@@ -350,21 +350,37 @@ olcs.FeatureConverter.prototype.olLineStringGeometryToCesium = function(layer, f
...
     positions: positions,
     width: this.extractLineWidthFromOlStyle(olStyle),
     vertexFormat: appearance.vertexFormat
-  });
+  };
+
+  var outlinePrimitive;
+  var heightReference = this.getHeightReference(layer, feature, olGeometry);
+
+  if (heightReference == Cesium.HeightReference.CLAMP_TO_GROUND) {
```
Comment (gberaudo) [inline src/featureconverter.js]: If the positions are 3D (just test the first one), could you disregard the `CLAMP_TO_GROUND` height reference?

Comment (oterral) [inline src/featureconverter.js]: You mean if the coordinates is x,y,z. we deactivate the clampToGround?

Comment (gberaudo) [inline src/featureconverter.js]: yes

Comment (oterral) [inline src/featureconverter.js]: We don't do this for polygons why doing it for lines ? and that doesn't make sense if the user specify clampToGround  he expects the dtat are clamped to ground whatever the z value.

Comment (gberaudo) [inline src/featureconverter.js]: OK, I checked the KML documentation and they do like you propose: https://developers.google.com/kml/documentation/altitudemode. So OK to merge. However, please update the README.md.

**Comment by gberaudo** [comment]
Thanks @oterral.

**Comment by oterral** [comment]
Thx for the review @gberaudo

**Pull request merged**